app/models/foreman_tasks/concerns/action_triggering.rb

```diff
@@ -35,11 +35,11 @@ def destroy
         dynflow_task_wrap(:destroy) { super }
       end
 
-      def update_attributes(*args)
+      def update(*args)
```

Comment:
I think we should still do aliasing here:

```
alias_method :update_attributes, :update
```

It seems the `update_attributes` is not actually deprecated in Rails and calling `obj.update_attributes` doesn't trigger this method.

Reply:
hrm... I would have thought https://github.com/rails/rails/blob/375a4143cf5caeb6159b338be824903edfd62836/activerecord/lib/active_record/persistence.rb#L432 would take care of this. :man_shrugging:

Reply:
That's how ruby works unfortunately. Seems like rubocop prefers `alias` over `alias_method` (don't ask me why :)